docs/language/overview.md

```diff
@@ -1060,11 +1078,11 @@ where a `<spec>` has one of three forms:
 The first form is a customary colon-separated field and value similar to JavaScript,
 where `<field>` may be an identifier or quoted string.
 The second form is an implied field reference `<ref>`, which is shorthand
-for `<ref>:<ref>`.  The third form is the `...` spread operator which expects
+for `<ref>:<ref>` (XXX).  The third form is the `...` spread operator which expects
```

Comment:
@nwt: Yes, `<field>` instead of `<ref>` definitely improves it for me, so I've pushed that change. I also realize now that when highlighting this text, I'd already forgotten that there was separate section already in the doc that explicitly describes what an "implied field reference" is, so I added a link to that as well.

I think it still leaves open the question of how to briefly describe what it's shorthand for. As it now stands, I've got it as "...which is shorthand for `<field>:<ref>`." In my mind, instead of `<ref>` it's be great if there's a way it could effectively say `<the value associated with <field>>` but I know that's wordy and awkward, so I'm happy with `<ref>` if you are. :smile: